client/my-sites/plan-features/index.jsx

```diff
@@ -679,6 +679,14 @@ export class PlanFeatures extends Component {
 		const description = feature.getDescription
 			? feature.getDescription( abtest, this.props.domainName )
 			: null;
+
+		const recordEvent = feature.getTrackingLabel
+			? () =>
+					this.props.recordTracksEvent( 'calypso_plan_link_click', {
+						label: feature.getTrackingLabel(),
```

Comment:
Should we be adding the plan slug here as well?

Reply:
I have to admit I'm not totally convinced we should have the label here.

I think there may be a better way to describe the link location in order to differentiate if from other links. E.g. - as @enejb pointed out - it could use a plan slug and maybe a static label that defines where the link is located, e.g. `plan_features_list_item`.

Reply:
The eventual use of that event is to identify where people leave the page. Let's say there is one of those links that attracts a lot of clicks, it would be interesting to find out why.
To do that, I need to know the string people are attracted to, more than where the link was or it was contextual to. That's why I indicated the label as a property.

Reply:
Thank you for the clarification 👍